front-end/linux-help.html

```diff
@@ -42,9 +42,10 @@
             </p>
             <p class="code">
                 sudo usermod -a -G dialout $USER
+		sg dialout "npm start"
```

Comment:
Maybe we should leave this line out then because there could be different groups and I'm unsure what command we would run... maybe "flasher.js" but I don't  know to be precise. I guess people can log in and out quickly